src/instance/mod.rs

```diff
@@ -18,27 +18,39 @@ use data::{Map,Monster};
 
 pub mod management;
 
+// Computation of the duration of one so-called 'tick'. A 'tick' is a
+// unite in the discrete time of lycan. By default, its value is set
+// to about 16ms.
```

Comment:
@Vaelden : I just had an idea that you may like. How does it sound to run lycan on a Raspberry Pi? =) With a tick around 1Hz, you would be able to serve many more players than with 60Hz.